files/etc/uci-defaults/20-realtek-poe-migrate-poe-type

```diff
@@ -0,0 +1,23 @@
+[ ! -s /etc/config/poe ] && exit 0
+
+. /lib/functions.sh
+
+function migrate_port_poe_type() {
+    local cfg="$1"
+    local poe_plus poe_type
+
+    config_get poe_plus "$cfg" poe_plus ""
+    config_get poe_type "$cfg" poe_type ""
+
+    # Only translate if not already configured
+    [ -z "$poe_type" ] && [ "$poe_plus" = "1" ] \
+        && uci set poe.$cfg.poe_type=ieee8023at
```

Comment:
Shouldn't `$cfg` be quoted?

Reply:
Would probably be safer, yes. `$cfg` contains values like "cfg012345", but we shouldn't make assumptions that will always be the case.